docs/release-train-issue-template.md

```diff
@@ -74,4 +74,10 @@ For minor or major releases:
 - [ ] Update version for [Lightbend Supported Modules](https://developer.lightbend.com/docs/lightbend-platform/introduction/getting-help/build-dependencies.html) in [private project](https://github.com/lightbend/lightbend-technology-intro-doc/blob/master/docs/modules/getting-help/examples/build.sbt)
 - [ ] Update [akka-dependencies bom](https://github.com/lightbend/akka-dependencies)
 - [ ] Update [Akka Guide samples](https://github.com/akka/akka-platform-guide)
+- [ ] Update quickstarts:
+  - [ ] [Akka HTTP Java](https://github.com/akka/akka-http-quickstart-java.g8/blob/main/src/main/g8/default.properties)
+  - [ ] [Akka HTTP Scala](https://github.com/akka/akka-http-quickstart-scala.g8/blob/main/src/main/g8/default.properties)
+  - [ ] [Akka gRPC Java](https://github.com/akka/akka-grpc-quickstart-java.g8/blob/main/src/main/g8/default.properties)
+  - [ ] [Akka gRPC Scala](https://github.com/akka/akka-grpc-quickstart-scala.g8/blob/main/src/main/g8/default.properties)
```

Comment:
Do we have to have an explicit dependency to Akka HTTP in the gRPC samples, or could that just be dependency via akka-grpc?

Reply:
It's currently explicit, but I wonder if it really is needed.

Reply:
Making transitive in:
https://github.com/akka/akka-grpc-quickstart-scala.g8/pull/72
https://github.com/akka/akka-grpc-quickstart-java.g8/pull/65